src/Cake.Issues.GitRepository/GitRepositoryIssuesProvider.cs

```diff
@@ -124,6 +129,44 @@ private IEnumerable<IIssue> CheckForBinaryFilesNotTrackedByLfs(IssueCommentForma
             return result;
         }
 
+         /// <summary>
+        /// Checks for files path length.
+        /// </summary>
+        /// <param name="length">Max length to be used to check the files.</param>
+        /// <returns>List of issues for repository files.</returns>
+        private IEnumerable<IIssue> CheckForFilesPathLength(int length)
+        {
+
+            IEnumerable<string> allFiles = this.GetAllFilesFromRepository();
+            List<IIssue> result = new List<IIssue>();
+
+            if (!allFiles.Any())
+            {
+                return new List<IIssue>();
+            }
+
+            foreach (string file in allFiles)
+            {
+                string message = null;
+                int pathLength = file.Length;
+
+                if (pathLength > length) {
+                    message = $"The path for the binary file \"{file}\", is too long";
+                }
+
+                var ruleDescription = new FilePathLengthRuleDescription();
+
+                result.Add(
```

Comment:
Issue should only be created if path is too long (inside the `pathLengt > length` block). This is also the reason for the failing test case

Reply:
You were quicker with your fix :)

Reply:
Fixed in previous commit.